### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by Simonliuwaterloo**
Some simple cosmetic changes. Didn't make any major changes.

Some notable changes:
* Changed "eventName" to "userInput"
* Removed some unused code and outdated comments.
* We are more conservative in terms of printing colors, now only use green for active states, use red for command not found error
* The output structure has changed, now it has three sections: logs after user input, current state hierarchy, list of all the commands, (maybe) env variable prompt, and the command prompt. Instead of asking user for event input, we simply ask for a "command" input, which can be a regular command or an event command, after every command input these sections will be printed:
image

### CODE REVIEW

File: org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm [hunk 1]
```diff
@@ -32,8 +32,6 @@ logging.addLevelName(SCLOG, "STATE")
 Reset = "\u001B[0m"
 Red = "\u001b[31m"
 Green = "\u001b[32m"
-Blue = "\u001B[34m"
```
Comment (AidanCampbell738) [inline org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm]: I don't agree with your changes to the colours here. What's wrong with blue and cyan for transitions and events? Also green and red was perfect for showing which states were active and inactive. 
The original colour scheme made the large amount of text output easier to process while also making it pleasing to look at. I think we should stick with what we have.

Comment (Simonliuwaterloo) [reply org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm]: Ok I will change it back

Comment (Simonliuwaterloo) [reply org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm]: On second thought, @Chestshire  @RuomeiYan what do you guys think?

**Review by Chestshire — APPROVED** [review]
LGTM.

**Review by AidanCampbell738 — APPROVED** [review]

**Pull request merged**